src/sdk/model/edge/edge.ts

```diff
@@ -436,6 +436,19 @@ export class Edge extends Entity {
       return new Task(json);
     });
   }
+
+  /**
+   * Update the edge interface.
+   * @param {EdgeInterface} request
+   * @returns {Promise<Task>}
+   */
+  /* istanbul ignore next: autogenerated */
+  async updateInterface(request: EdgeInterface): Promise<Task> {
```

Comment:
the request and response models aren't the same. Should accept a EdgeInterfaceUpdateRequest